build/Makefile

```diff
@@ -296,6 +296,8 @@ $(INTERMEDIATE_DIR)/$(OUTPUT_PACKAGE_PREFIX).tar : $(RUBY_DEST_DIR) $(IN_PLUGINS
 	cd $(DSC_DIR)/release; $(COPY) `ls omsconfig-*.ssl_100.*.deb | sort | tail -1` $(INTERMEDIATE_DIR)/100
 	cd $(INTERMEDIATE_DIR)/100; for f in omsconfig-*.{rpm,deb}; do SOURCE=$$f; DEST=`echo $$SOURCE | sed 's/.ssl_100././'` ; $(MV) $$SOURCE $$DEST; done
 
+	cd $(DSC_DIR)/release; $(COPY) `ls *.zip` $(TARGET_DIR)/
```

Comment:
No, I think this is wrong. $(TARGET_DIR) already contains an architecture designation, so you don't need it. I think it's redundant, on the build disk, to have something like omsagent/version/target_dir_with_arch/arch/<stuff>.

Please don't commit this yet, I'd like to take a closer look at this tomorrow morning. But this change is not right, not unless we decide to change the build structure in general.